WebHostLib/templates/multitracker__Celeste64.html

```diff
@@ -0,0 +1,91 @@
+{% extends "multitracker.html" %}
+{% block head %}
+    {{ super() }}
+    <script type="application/ecmascript" src="{{ url_for("static", filename="assets/jquery.scrollsync.js") }}"></script>
+    <script type="application/ecmascript" src="{{ url_for("static", filename="assets/lttpMultiTracker.js") }}"></script>
```

Comment:
Why is the Celeste tracker importing the Link to the Past JS?

Reply:
This file controls the same scroll for multiple tables. It's not used here because we have just one. Reference is removed.